### ISSUE TITLE
FTP IP not shown at some screen resolutions

### ISSUE THREAD

**Issue by cousteaulecommandant** [TeamAmaze/AmazeFileManager#1114]
Phone's IP is not shown on the FTP server screen on some locales (Spanish specifically, but I tried changing the language to English and it works).  It used to on previous versions (years ago).
(Nevertheless, when FTP is running, I get a notification in the status bar with the IP, so at least I can use that.) 

Android 7.1.1 
Phone: bq Aquaris U 
Amaze v3.2.1
Locale: Spanish (Spain) 

<img height="500px" src="https://user-images.githubusercontent.com/6584870/37874670-d42b6444-3033-11e8-88a5-1b043e012a23.png"/>

**Comment by EmmanuelMess** [comment]
It's working on master.

**Comment by cousteaulecommandant** [comment]
On what screen size?

Further specs:
Screen size: 5.0"
Screen resolution: 720x1280
Smallest width: 360 dp

I just tried changing "Smallest width" under *Developer options* from 360 to 433 and now it fits.
The problem seems to be that on such a narrow screen, something like *"Estado: Funcionando (ftp://
192.168.1.128)"* wraps to a second line, and the second line is not displayed.

(My apologies if this has been actually fixed in a newer commit; I have only tested the latest available Google Play version)
